app.py

```diff
@@ -3,6 +3,8 @@
 from flask.ext.script import Server, Manager
 from flask.ext.migrate import MigrateCommand
 
+app.debug = True
```

Comment:
You shouldn't need this, it's set to True in the config file that gets loaded, unless I'm missing something.